stable/pachyderm/README.md

```diff
@@ -20,6 +20,7 @@ The following table lists the configurable parameters of `pachd` and their defau
 | Parameter                | Description           | Default           |
 |--------------------------|-----------------------|-------------------|
 | `pachd.image.repository` | Container image name  | `pachyderm/pachd` |
+| `pachd.pfsCache`         | File System cache size| `0G`              |
 | `*.image.tag`            | Container image tag   | `<latest version>`|
```

Comment:
should the image.tag default here match the appVersion `1.6.7` ?   I notice in values you've set it to `1.6.7`.

Reply:
@paulczar @prydonius suggested that we should set it like this, so that we can avoid updating the README file everytime the chart is bumped.